Makefile

```
@@ -20,10 +20,14 @@ CI_FLAGS =\
 	DBT_LOG_FORMAT=json
 
 .PHONY: dev
-dev: ## Installs dbt-* packages in develop mode along with development dependencies.
+dev_req: ## Installs dbt-* packages in develop mode along with only development dependencies.
 	@\
 	pip install -r dev-requirements.txt -r editable-requirements.txt
 
```

This is the other part I didn't notice this the first time around.

```suggestion

.PHONY: dev
```